.github/workflows/ci.yml

```diff
@@ -45,54 +34,119 @@ jobs:
       - name: Build all packages
         run: npm run build
 
+  get-commands:
+    name: Get scaffold commands
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ubuntu-latest
+    outputs:
+      scaffoldCommands: ${{ steps.set-matrix.outputs.scaffoldCommands }}
+    steps:
+      - id: set-matrix
+        run: echo "scaffoldCommands=[{\"cmd\":\"generate-app\",\"hasBackend\":false},{\"cmd\":\"generate-app-backend\",\"hasBackend\":true},{\"cmd\":\"generate-panel\",\"hasBackend\":false},{\"cmd\":\"generate-datasource\",\"hasBackend\":false},{\"cmd\":\"generate-datasource-backend\",\"hasBackend\":true},{\"cmd\":\"generate-scenes-app\",\"hasBackend\":false}]" >> $GITHUB_OUTPUT
```

Comment:
I am not sure I understand why you want this to be generated with a bash command. I don't see anything dynamic?

Reply:
Thanks for questioning this. I hadn't really put much thought into doing it any other way. Maybe it would make more sense if we read from a json file?

`echo "scaffoldCommands=$(jq -c . < ./scaffoldCommands.json)" >> $GITHUB_OUTPUT`

Reply:
can't you simply define the matrix as?:

```
    strategy:
      matrix:
        cmd: [generate-app, generate-app-backend, generate-panel, generate-datasource, generate-datasource-backend, generate-scenes-app]
        hasBackend: [true, false]
```

I guess you don't want weird combinations like `cmd: generate-datasource-backend, hasBackend: false` so I think you can also do:

```
    strategy:
      matrix:
        include:
        - cmd: generate-app
          hasBackend: false
        - cmd: generate-app-backend
          hasBackend: true
          ...
```
according to https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations